tests/common/templates/pfc_storm_stop_sonic.j2

```diff
@@ -1,6 +1,6 @@
 cd {{pfc_gen_dir}}
 {% if (pfc_asym is defined) and (pfc_asym == True) %}
-nohup sh -c "{% if pfc_storm_stop_defer_time is defined %}sleep {{pfc_storm_stop_defer_time}} &&{% endif %} sudo pkill -f 'python {{pfc_gen_file}} {% if pfc_gen_multiprocess is defined %}-m {% endif %}-p {{pfc_queue_index}} -t 65535 -n {{pfc_frames_number}} -i {{pfc_fanout_interface}}'" > /dev/null 2>&1 &
+nohup sh -c "{% if pfc_storm_stop_defer_time is defined %}sleep {{pfc_storm_stop_defer_time}} &&{% endif %} sudo pkill -f 'python3 {{pfc_gen_file}} {% if pfc_gen_multiprocess is defined %}-m {% endif %}-p {{pfc_queue_index}} -t 65535 -n {{pfc_frames_number}} -i {{pfc_fanout_interface}}'" > /dev/null 2>&1 &
```

Comment:
Will this affect non-T2 devices?
My only concern is whether all DUTs have 'python3' command

Reply:
@yejianquan sorry good pick, the stop file needs to be modified on t2. Fixed